python/sqlflow_submitter/db.py

```diff
@@ -57,7 +57,7 @@ def parseHiveDSN(dsn):
 
 def parseMaxComputeDSN(dsn):
     # access_id:access_key@service.com/api?curr_project=test_ci&scheme=http
-    user_passwd, address, config_str = re.findall("^(.*)@([.a-zA-Z0-9/]*)(\?.*)?", dsn)[0]
+    user_passwd, address, config_str = re.findall("^(.*)@([-.a-zA-Z0-9/]*)(\?.*)?", dsn)[0]
```

Comment:
Also update the testcase to fit this fix?

Reply:
Done.